fastai/transforms.py

```diff
@@ -380,6 +389,8 @@ def do_transform(self, x, is_y):
 
 
 class RandomDihedral(CoordTransform):
+    """ Rotates images by random multiples of 90 degrees. 
```

Comment:
Nearly but not quite! There are 8 symmetries of a square (the [dihedral group of order 8](https://groupprops.subwiki.org/wiki/Dihedral_group:D8)), but only 4 90 degree rotations. So you also need to mention the flip! :)